src/managed/dir.proj

```diff
@@ -3,11 +3,15 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
 
-  <Target Name="Build">
+  <Target Name="Build"
+          DependsOnTargets="GetLatestCommitHash">
     <MakeDir  Condition="!Exists('$(pOutDir)')" Directories="$(pOutDir)" />
 
     <PropertyGroup>
       <Args>/p:Configuration=$(ConfigurationGroup)</Args>
+      <_EnvironmentVariables>NUGET_PACKAGES=$(PackagesDir)</_EnvironmentVariables>
```

Comment:
We really need to get out of the this business of relying on environment variables. It makes things a lot harder on ourselves.